client/src/App.jsx

```diff
@@ -2,26 +2,47 @@ import { lazy, Suspense } from "react";
 import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom";
 import { ErrorBoundary } from "react-error-boundary";
 import NavigationBar from "./components/navbar/NavigationBar";
-
 import { ErrorState, LoadingState } from "./components/states";
 
 const Homepage = lazy(() => import("./pages/home/Homepage"));
 const Events = lazy(() => import("./pages/events/Events"));
 const Projects = lazy(() => import("./pages/projects/Projects"));
 const Resources = lazy(() => import("./pages/resources/Resources"));
+const Login = lazy(() => import("./pages/manager/login/Login"));
+
+function Client() {
+	return (
+		<>
+			<NavigationBar />
+			<Routes>
+				<Route path="/homepage" element={<Homepage />} />
+				<Route path="/activity" element={<Activity />} />
```

Comment:
Produces error, "Activity is not defined." The website won't load without commenting this out (which prevents access to the Events and Projects pages too).

Reply:
I removed activity route. It's deprecated